sklearn/linear_model/ridge.py

```diff
@@ -867,10 +871,12 @@ def fit(self, X, y, sample_weight=None):
             #fit_params = {'sample_weight' : sample_weight}
             fit_params = {}
             gs = GridSearchCV(Ridge(fit_intercept=self.fit_intercept),
-                              parameters, fit_params=fit_params, cv=self.cv)
+                              parameters, fit_params=fit_params, cv=self.cv,
+                              scoring = self.scoring)
```

Comment:
I think RidgeCV with cv!=None should have the same outcome as using GridSearchCV if they are specified same `cv`. Or I misunderstand what you mean?